pkg/ocm/helpers.go

```diff
@@ -83,6 +83,7 @@ func IsValidClusterName(clusterName string) bool {
 
 func ClusterNameValidator(name interface{}) error {
 	if str, ok := name.(string); ok {
+		str := strings.Trim(str, " \t")
```

Comment:
Ended up doing in this manner, I looked into the survey dependency and they have a TransformString option which would have been useful, but it only applies transform after validation, so we would either have to allow ' \t' on the regex or trim the internal string anyway

Reply:
@vkareh PTAL

Reply:
strings.Trim is fine